src/addremove.js

```diff
@@ -0,0 +1,84 @@
+const Task = [] || JSON.parse(localStorage.getItem('Task'));
+
+const TaskList = document.querySelector('.to-do-list');
+
+const createElement = (e) => {
+  const InsertedDiv = document.createElement('div');
+  const InsertedCheckBox = document.createElement('input');
+  const toDoTask = document.createElement('input');
+  const deleteButton = document.createElement('button');
+
+  deleteButton.classList.add('remove-task');
+  deleteButton.innerHTML = '<i class=" fa-solid fa-trash-can"></i>';
+
+  InsertedDiv.classList.add('dynamic-Elements');
+
+  InsertedCheckBox.type = 'checkbox';
+  InsertedCheckBox.checked = e.complete;
+
+  toDoTask.value = e.description;
+  toDoTask.classList.add('task-layout');
+
+  InsertedDiv.setAttribute('index_id', e.index);
+  InsertedDiv.append(InsertedCheckBox, toDoTask, deleteButton);
+
+  TaskList.appendChild(InsertedDiv);
+};
+
+const addTask = (Task) => {
+  Task.forEach((element) => {
+    createElement(element);
+  });
+};
+
+const localStorageTasks = (Task) => {
+  window.localStorage.setItem('Task', JSON.stringify(Task));
+};
+
+const getLocalStorage = () => {
+  const theTask = window.localStorage.getItem('Task');
+
+  if (theTask !== null) {
+    addTask(JSON.parse(theTask));
+  }
+
+  return theTask;
+};
+
+const addToDoTask = (value) => {
+  const task = {
+    description: value,
+    completed: false,
+    index: Task.length + 1,
+  };
+
+  Task.push(task);
+  localStorageTasks(Task);
```

Comment:
- [ ] When I try to add new todos, they are not saved instantly into the local storage as shown here until i close and reopen the local storage 👇 

![image](https://user-images.githubusercontent.com/63473584/178100838-e999ab55-f9b4-4000-bccc-1bcb4b9208d0.png)
Kindly check it out.